### ISSUE TITLE
Pretrained Weights

### ISSUE THREAD

**Issue by danielgordon10** [ppwwyyxx/GroupNorm-reproduce#2]
Would it be possible for you to share pretrained weights for resnet GN? Transferring directly from the pytorch BN weights gives very bad accuracy.

**Comment by ppwwyyxx** [comment]
Why would weights of a different model (with BN) work?
Pretrained weights in tensorflow was mentioned in https://github.com/ppwwyyxx/GroupNorm-reproduce/tree/master/ImageNet-ResNet-TensorFlow

**Comment by ppwwyyxx** [comment]
The instructions are now updated to include the command to evaluate the above model on ImageNet.

**Comment by danielgordon10** [comment]
> Why would weights of a different model (with BN) work?

I wasn't expecting them to, but they're the only weights I could find. Especially for smaller models like ResNet18.

Do you happen to have pytorch weights? I can convert the TF weights if necessary, but I was trying to follow your Pytorch code.

**Comment by ppwwyyxx** [comment]
There are also caffe2 weights at https://github.com/facebookresearch/Detectron/tree/master/projects/GN#pretrained-models-with-gn, but I don't have pytorch weights at the moment.

**Comment by ppwwyyxx** [comment]
A pytorch model was uploaded at https://github.com/ppwwyyxx/GroupNorm-reproduce/releases/tag/v0.1
